pkg/utils/test/bpf_map.go

```diff
@@ -31,6 +31,7 @@ import (
 type CleanupFn func()
 
 func InitBpfMap(t *testing.T, config options.BpfConfig) (CleanupFn, *bpf.BpfLoader) {
+	bpf.CleanupBpfMap()
```

Comment:
why do we need this? Any test forget to cleanup before exit?

Reply:
sorry... I forget to remove, just for test.